vnsitimer.c

```diff
@@ -449,7 +449,12 @@ void CVNSITimers::Action()
             if (event->EndTime() < time(nullptr))
               continue;
 
-            std::string title(event->Title());
+            const char *p = event->Title();
+
+            if (!p)
+              continue;
```

Comment:
I had a look at surrounding code. Would it be actually possible to replace your changes completely by moving a

```
if (event->Title() == nullptr)
  continue
```
above the `std::string title(event->Title());` line? We should have a "read lock" in this loop, right?

Reply:
@M-Reimer, the commit is updated.